shakedown/dcos/command.py

```diff
@@ -39,7 +39,12 @@ def run_command(
     key = validate_key(key_path)
 
     transport = get_transport(host, username, key)
-    transport = start_transport(transport, username, key)
+
+    if transport:
+        transport = start_transport(transport, username, key)
+    else:
+        print('error: unable to connect to ' + host)
```

Comment:
Sure; that will make this a little bit larger of a PR but seems like a good chance to introduce more code consistency.